tests/test_fmi.py

```diff
@@ -1117,6 +1117,17 @@ def test_atol_auto_update6(self):
         model.simulate(options=opts, algorithm=NoSolveAlg)
         np.testing.assert_allclose(opts["CVode_options"]["atol"], [0.03, 0.03])
 
+    @pytest.mark.parametrize("atol", [1e-4, [1e-4], np.array([1e-4])])
```

Comment:
Maybe also np.array(1e-4) for completeness?

Reply:
Added, see latest commit. Required some code change since 0-dim numpy arrays are a surprising mess.